WebApp/src/components/StudyGroup/StudyGroup.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Toolbar from '../Toolbar/Toolbar';
+import Button from '@material-ui/core/Button';
+import NavigationIcon from '@material-ui/icons/Navigation';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Input from '@material-ui/core/Input';
+import './StudyGroup.scss';
+
+class StudyGroup extends React.Component {
+  state = {
+    sidebarOpen: true,
+    group: ''
+  }
+
+  Submit = () => {
+    this.props.history.push('/timetable');
+  }
+
+  handleSubmit = (evt) => {
+    this.setState({ group: evt.target.value });
+  }
+
+  canBeSubmitted = () => {
+    const group = this.state.group;
+    return (
+      group != ''
+    );
+
+  }
+
+  render() {
+    const valid = this.canBeSubmitted();
+    return (
+      <div className="container home">
+        <Toolbar />
+        <div className="StudyGroup-wrapper">
+          <FormControl
+            fullWidth={true}
```

Comment:
When component's prop accepts boolean value and that value is true, you can omit it. Same as in HTML: `<div hidden>can you see me?</div>` instead of `<div hidden="true">`